courses/free-courses-en.md

```diff
@@ -967,6 +967,7 @@
 
 #### React
 
+* [Complete React course for beginner](https://youtube.com/playlist?list=PLRAV69dS1uWQos1M1xP6LWN6C-lZvpkmq&si=Mu6nucz4_62aP3V0) - Hitesh Choudhary
```

Comment:
```suggestion
* [Complete React course for beginner](https://youtube.com/playlist?list=PLRAV69dS1uWQos1M1xP6LWN6C-lZvpkmq) - Hitesh Choudhary
```
entire repo is marked correctly for Hacktoberfest

Reply:
I have made all seggested changes in the course link.